core/modules/views/templates/views-view-list.tpl.php

```diff
@@ -3,8 +3,17 @@
  * @file
  * Default simple view template to display a list of rows.
  *
- * - $title : The title of this group of rows.  May be empty.
+ * - $title: The title of this group of rows.  May be empty.
+ * - $rows: @todo document.
+ * - $row_classes: @todo document.
+ * - $list_type_prefix: @todo document.
+ * - $list_type_suffix: @todo document.
+ * - $wrapper_prefix: @todo document.
+ * - $wrapper_suffix: @todo document.
+ *
+ * Additional variables:
  * - $options['type'] will either be ul or ol.
+ *
```

Comment:
Can we separate this out into a different PR? This doesn't seem related to the issue at hand.